models/models.js

```diff
@@ -17,11 +17,28 @@ exports.readAllApis = () => {
     return fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
         .then((data) => {
             const parsedData = JSON.parse(data);
-            console.log(parsedData)
             return parsedData;
         });
 };
 
+exports.retrieveArticles = () => {
+    return db.query("SELECT article_id, article_img_url, author, created_at, title, topic, votes FROM articles ORDER BY created_at DESC;");
+};
+
+exports.articleComments = () => {
+    return db.query("select count(*) from comments join articles on comments.article_id = articles.article_id group by articles.article_id order by articles.created_at desc;").then(({ rows }) => {
+        return rows
+    })
+};
+
+exports.lookupObj = () => {
+    return db.query("SELECT article_id, count(*) from comments group by article_id;").then(({ rows }) => {
+        const lookup = rows
+        return lookup
+    })
+}
```

Comment:
All the logic for making one combined function to retrieve your comment count is here, you just have to join them all together